File: Source/Examples/DocumentationExamples/HelloWorld/WpfApplication1/MainViewModel.cs
namespace WpfApplication1
{
    using System;

    using OxyPlot;
    using OxyPlot.Series;

    public class MainViewModel
    {
        public MainViewModel()
        {
            this.MyModel = new PlotModel("Example 1");
            this.MyModel.Series.Add(new FunctionSeries(Math.Cos, 0, 10, 0.1, "cos(x)"));
        }

        public PlotModel MyModel { get; private set; }
    }
}

File: Source/Examples/WPF/SimpleDemoFsharp/MainViewModel.fs
namespace SimpleDemoFsharp

open OxyPlot
open OxyPlot.Series

type MainViewModel() =
    let myModel = PlotModel()
    do
        myModel.Series.Add(FunctionSeries(cos, 0.0, 10.0, 0.1, "cos(x)"))
    member mainWindow.MyModel with get() = myModel